corehq/apps/hqmedia/static/hqmedia/js/hqmediauploaders.js

```diff
@@ -1,15 +1,19 @@
-/* globals HQMediaUploaderTypes */
-hqDefine("hqmedia/js/hqmediauploaders", function () {
+hqDefine("hqmedia/js/hqmediauploaders",[
+    "hqwebapp/js/initial_page_data",
+    "underscore",
+    'file-uploader',
+], function (intialPageData, _, HQMediaUploaderTypes) {
```

Comment:
`intialPageData` => `initialPageData`

Reply:
(throughout the PR)

Reply:
I'm also going to be strict and say we should make these dependency lists as consistent as possible, so that they become basically boilerplate and anything unusual sticks out. That means reformatting to put one parameter per line, and reordering the dependencies at least to put the major global libraries (jquery, knockout, underscore) at the top.